lib/OnyxUtils.ts

```diff
@@ -1240,12 +1269,14 @@ function updateSnapshots(data: OnyxUpdate[], mergeFn: typeof Onyx.merge): Array<
  * @param collection Object collection keyed by individual collection member keys and values
  * @param mergeReplaceNullPatches Record where the key is a collection member key and the value is a list of
  * tuples that we'll use to replace the nested objects of that collection member record with something else.
+ * @param retryAttempt retry attempt
  */
 function mergeCollectionWithPatches<TKey extends CollectionKeyBase, TMap>(
     collectionKey: TKey,
     collection: OnyxMergeCollectionInput<TKey, TMap>,
     mergeReplaceNullPatches?: MultiMergeReplaceNullPatches,
-    isProcessingCollectionUpdate = false,
+    isProcessingCollectionUpdate?: boolean,
```

Comment:
Why did you remove the default value for this parameter if we don't pass the nullish value anywhere? I think having default `false` is fine

This would prevent the need for `!!isProcessingCollectionUpdate` [in line 1317](https://github.com/Expensify/react-native-onyx/pull/694/files#diff-c584aabafbadccee45af6a779a6e26057cf5b2546ed0de9d69a8bc9ac1bae791R1317) and make it more readable